docs/pages/es/project/security-vulnerabilities.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: page
+title: Vulnerabilidades de seguridad
+permalink: es/project/security-vulnerabilities
+description: Cómo el equipo de Meshery maneja las vulnerabilidades de seguridad.
+---
+
+# Informar una vulnerabilidad
+Estamos muy agradecidos con los investigadores de seguridad y los usuarios
```

Comment:
Change 'Estamos muy agradecidos con los investigadores de seguridad y los usuarios que informan sobre las vulnerabilidades de seguridad de Meshery' to
'Estamos muy agradecidos con quienes investigan temas de seguridad y quienes informan sobre las vulnerabilidades de seguridad de Meshery.'